src/test/regress/expected/subquery_and_cte.out

```diff
@@ -246,8 +247,7 @@ FROM
 	     ) SELECT * FROM cte ORDER BY 1 DESC
      ) as foo
 ORDER BY 1 DESC;
-DEBUG:  generating subplan XXX_1 for CTE cte: SELECT DISTINCT users_table.user_id FROM subquery_and_ctes.users_table, subquery_and_ctes.events_table WHERE ((users_table.user_id OPERATOR(pg_catalog.=) events_table.user_id) AND (events_table.event_type OPERATOR(pg_catalog.=) ANY (ARRAY[1, 2, 3, 4])))
-DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT user_id FROM (SELECT cte.user_id FROM (SELECT intermediate_result.user_id FROM read_intermediate_result('XXX_1'::text, 'binary'::citus_copy_format) intermediate_result(user_id integer)) cte ORDER BY cte.user_id DESC) foo ORDER BY user_id DESC
+DEBUG:  CTE cte is going to be inlined via distributed planning
```

Comment:
For example this one expects the final query tto be a router query, which means the CTE should be materialized

Reply:
ping on this